File: C/URI/1079.c
#include <stdio.h>
 
int main() {
 
 double a, b, c, media;
  int n;
        scanf("%d", &n);
        while(n > 0)
        {
                scanf("%lf %lf %lf", &a, &b, &c);
                media = ((a * 2) + (b * 3) + (c * 5))/10;
                printf("%.1lf\n", media);
        n = n - 1;


        }

    return 0;
}

File: php/URI/1079.php
<?php

$num = (int)readline();

for ($i = 0; $i < $num; $i++) {
    
    $l = explode(' ', readline());

    $media = ($l[0]*2 + $l[1]*3 + $l[2]*5)/10;

    printf("%.1f\n", $media);
}



?>